### PR TITLE
[GCI-142] Lambdas and Method references instead of anonymous classes

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/GCI-142
Related PR: #2366

It is cleaner way to deal with anonymous classes since JDK8 so why not use it? :)

### CODE REVIEW

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa Can you look at these old PR's that are not yet merged? :)
#2407 
#2398 
#2385 
#2410

**Comment by dkayiwa** [comment]
Sure let me do so. :)

File: api/src/main/java/org/openmrs/Concept.java [hunk 1]
```diff
@@ -886,7 +886,7 @@ public ConceptName getShortNameInLocale(Locale locale) {
 			}
 		} else {
 			shortNames = getNames().stream()
-							.filter(n -> n.isShort())
+							.filter(ConceptName::isShort)
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/Concept.java]: How does this change make the code better?

Comment (BartlomiejRasztabiga) [reply api/src/main/java/org/openmrs/Concept.java]: Method references are believed to be more readable, because they remove redundant "n ->" :)

**Comment by dkayiwa** [comment]
Thanks @BartlomiejRasztabiga 😊

**Pull request merged**